DevHome.sln

```diff
@@ -163,6 +163,16 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "WSLExtension", "WSLExtensio
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WSLExtension", "extensions\WSLExtension\WSLExtension.csproj", "{B6153EEA-EADE-4BAA-B47D-6B48205C6696}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FileExplorerGitIntegration", "extensions\GitExtension\FileExplorerGitIntegration\FileExplorerGitIntegration.csproj", "{5366F178-AD59-475C-B78D-2E6483313F9E}"

```

Comment:
I've noticed a few files in the addition are suffering from this. I'm not sure which settings are responsible, but it might be worth looking into how that's happening.

Reply:
I fixed a batch of what I could find. Do you see any others?

Reply:
I have a .gitattributes file that from what I understand *should* be normalizing this stuff but it isn't. Nor does the rule in .editorconfig seem to force new files to use crlf like it should. I've gone down this rabbit hole a few times and never figured it out.

In any case, it's just nice to check these in the first time with crlf so we don't have to think about it again. I don't see any new files with lf so I think we're good here now.